skglm/penalties/separable.py

```diff
@@ -466,3 +466,48 @@ def is_penalized(self, n_features):
     def generalized_support(self, w):
         """Return a mask with non-zero coefficients."""
         return w != 0
+
+
+class PositiveConstraint(BasePenalty):
+    """Non-negative least square penalty."""
```

Comment:
Since we changed the name, we better put ``Positive constraint penalty.`` ?

Reply:
```suggestion
    """Positivity constraint penalty."""
```